tree/30_generic_methods/file_copy_from_local_source.cf

```diff
@@ -28,6 +28,11 @@
 
 bundle agent file_copy_from_local_source(source, destination)
 {
+  vars:
+     "class_prefix"        string => canonify("file_copy_from_local_source_${destination}");
+     "new_class_prefix"    string => canonify(join("_", "this.callers_promisers"));
+
   methods:
-      "placeholder" usebundle => file_copy_from_local_source_recursion("${source}", "${destination}", "0");
+      "copy without recursion" usebundle => file_copy_from_local_source_recursion("${source}", "${destination}", "0");
+      "new result classes"     usebundle => _classes_copy("${class_prefix}", "${new_class_prefix}");
```

Comment:
i'm not sure of the result here.
If you call twice file_copy_from_local_source_recursion, with same destination, you won't be able to differenciate the results (the class_prefix will be the same)